p24/src/refs.rs

```diff
@@ -0,0 +1,39 @@
+pub fn f1(s: &mut (u32, u32), flag: bool) -> &mut u32 {
+    match flag {
+        false => return &mut s.0,
+        true => return &mut s.1,
```

Comment:
You do not need explicit returns. Follow the Clippy lints (this also why you CI fails).